solutions/devicesimulation-nohub/single-vm/setupWrapper.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Copyright (c) Microsoft. All rights reserved.
+# Note: Windows Bash doesn't support shebang extra params
+set -ex
+
+while [ "$#" -gt 0 ]; do
```

Comment:
You don't need to duplicate this arguments in two scripts. You should be able to pass all arguments as a whole.

Reply:
What was the way you are recommending ? Could you elaborate ? 
This way I can keep this format --<name> <param> which is more readable than using $@

Reply:
Looked into this a bit more.. updated all the args as whole.